practices/development.md

```diff
@@ -74,6 +74,32 @@ The Sprint Board is broken down into these columns:
 
 In addition, we have a few other pieces of metadata to signal different kind of actions that would be needed 
 
+
+(coordination:team-syncs)=
+## Daily team syncs
+
+Throughout the week we have a lightweight asynchronous team sync process so that we can get on the same page.
+The primary goals of this process are:
+
+- To ensure that nobody is stuck on something
+- To signal-boost requests for review and help
+- To provide accountability of what we've been up to
+- To help us coordinate what to do next
+
+We use [the Geekbot](https://geekbot.com/) to manage this process.
+During a sync, each team member gets a message in their morning time with a few questions.
```

Comment:
I noticed that this Monday's bot message was at 7PM and not in the morning as usual.

Reply:
I wonder if that was just because I flipped the switch at that time and it sent a backlog of messages? Let's see what happens on Wednesday

Reply:
Yes, the Wednesday message was at 10 AM :tada: